client/src/redux/features/auth/authSlice.js

```diff
@@ -16,9 +16,9 @@ const authSlice = createSlice({
   name: 'auth',
   initialState,
   reducers: {
-    setUser: (state, action) => {
-      state.user = action.payload;
```

Comment:
Why did you delete this reducer?
And remove all the redux logic in sign up since the user will not be verified.